pyproject.toml

```diff
@@ -0,0 +1,46 @@
+[build-system]
+requires = [
+  "setuptools>=42",
+  "setuptools_scm[toml]>=6.2",
+]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools_scm]
+
+[project]
+name = "peek_data"
+description = "Live Data Reduction Dashboard"
+license = { "file" = "LICENSE" }
+readme = "README.md"
+classifiers = [
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: BSD License",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+]
+requires-python = ">=3.8"
+dynamic = ["version"]
+dependencies = []
+
+[project.urls]
+    "Documentation" = ""
+    "Bug Tracker" = "https://github.com/scipp/peek-data/issues"
+    "Source" = "https://github.com/scipp/peek-data"
+
+[tool.bandit]
+# Excluding tests because bandit doesn't like `assert`.
+exclude_dirs = ["tests"]
+
+[tool.codespell]
+skip = "./.git,./.tox,*/*_cache"
+
+[tool.isort]
+skip_gitignore = true
+line_length = 88
+multi_line_output = 2  # for compatibility with yapf
```

Comment:
Is this needed with `black`?

Reply:
No, it should simply be
```toml
[tool.isort]
skip_gitignore = true
profile = "black"
```

Reply:
`isort` sorts the imported libraries in the alphabetical order as well.
`black` and `isort` have some conflicts and it's described here.
https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html

Actually, I don't like that it re-orders the imported calls or classes in alphabetical order.
Can I just drop the `isort`??

Reply:
We use it in all other projects, I think alphabetical has some utility, so I think we should keep it.